### PR TITLE
queued vs running job webapp

### PR DESCRIPTION
**Pull request by Ishankoradia**
To be merged/deployed with backend https://github.com/DalgoT4D/DDP_backend/pull/562

### CODE REVIEW

File: src/components/Connections/Connections.tsx [hunk 1]
```diff
@@ -140,11 +145,13 @@ const getSourceDest = (connection: Connection) => (
...
   const toastContext = useContext(GlobalContext);
-  // const [blockId, setBlockId] = useState<string>('');
   const [connectionId, setConnectionId] = useState<string>('');
-  // const [syncingBlockId, setSyncingBlockId] = useState<string>('');
-  const [syncingConnectionId, setSyncingConnectionId] = useState<string>('');
-  const [syncLogs, setSyncLogs] = useState<Array<string>>([]);
+  const [syncingConnectionIds, setSyncingConnectionIds] = useState<
+    Array<string>
+  >([]);
+  // const [syncLogs, setSyncLogs] = useState<Array<string>>([]);
```
Comment (mdshamoon) [inline src/components/Connections/Connections.tsx:152]: Remove if not needed

**Review by mdshamoon — COMMENTED** [review]
Code wise seems fine. Have not done functional testing since I am unable to run backend on my local.

**Comment by Ishankoradia** [comment]
@mdshamoon  the staging is now in the branch of `559-distinguish-between-running-and-queued-jobs---backend`.
Could review this please thoroughly ? we will need to merge both together (frontend & backend)

**Comment by mdshamoon** [comment]
@Ishankoradia Looks really weird When the queuing action happens.

https://github.com/DalgoT4D/webapp/assets/32592458/eefd0bcd-b057-4aa5-a6da-82ebb99f7b51

I think just update the sync button with an icon and text. Waiting in queue. Also the transition looks really weird.
The waiting icon does not make sense under last run or last sync.

**Comment by Ishankoradia** [comment]
@mdshamoon could you check, have made the changes we discussed in the morning. 
I will fix the test cases in the meantime. Also if you could help me with the icon for `queue`. Thanks.

**Review by mdshamoon — APPROVED** [review]

**Pull request merged**